.eslintignore

```diff
@@ -1 +1,3 @@
 coverage
+node_modules
+support
```

Comment:
@Amir-61 I only see that one file under `support`, is there suppose to be other stuff there?